pkgs/development/compilers/bambu/default.nix

```diff
@@ -0,0 +1,250 @@
+{ runCommand
+, makeWrapper
+, buildEnv
+, fetchFromGitHub
+, stdenv
+, lib
+, pkgsi686Linux
+, libmpc
+, gmp
+, mpfr
+, boost
+, libxml2
+, xz
+, zlib
+, icu
+, suitesparse
+, mpfi
+, glpk
+, autoconf
+, autoconf-archive
+, automake
+, libtool
+, bison
+, doxygen
+, flex
+, graphviz
+, verilog
+, verilator
+, gmpxx
+, buddy
+, bash
+
+, enableClang7 ? false
+, enableClang8 ? false
+, enableClang10 ? false
+, enableClang11 ? false
+, enableClang12 ? false
+, enableClang13 ? false
+, # At least one clang version must be enabled.
+  enableClang16 ? true
```

Comment:
Why?

Reply:
These versions are used as frontends for processing C/C++ input by bambu. Bambu supports one compiler per supported major version of Clang or GCC.

I replaced the flags with a list of compilers